include/NAS2D/Renderer/Renderer.h

```diff
@@ -13,145 +13,147 @@
 #include "Color.h"
 #include "Point.h"
 #include "Rectangle.h"
-#include "../Signal.h"
-#include "../Resources/Image.h"
-#include "../Resources/Font.h"
 
 #include <string>
 
-namespace NAS2D {
+#include "../Resources/Font.h"
+#include "../Resources/Image.h"
+#include "../Signal.h"
 
-/**
- * \class Renderer
- * \brief Renderer base class.
- *
- * Provides a standard Renderer interface. The base Renderer can be used
- * but will act as a NULL interface.
- */
-class Renderer
+namespace NAS2D
 {
-public:
-	Renderer() = default;
-	Renderer(const Renderer& rhs) = default;
-	Renderer& operator=(const Renderer& rhs) = default;
-	Renderer(Renderer&& rhs) = default;
-	Renderer& operator=(Renderer&& rhs) = default;
-	virtual ~Renderer();
 
-	const std::string& driverName();
+	/**
+	 * \class Renderer
+	 * \brief Renderer base class.
```

Comment:
I've noticed not indenting for the namespace is pretty consistent across the project. If we're going to change that, we should probably open an issue for it first and let Leeor weigh in on the large formatting changes that would entail.